cmd/root.go

```diff
@@ -32,6 +34,7 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.trento.yaml)")
+	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "", "then minimum severity (error, warn, info, debug) of logs to output")
```

Comment:
I think you could put the default value as the 3rd parameter here, instead of a an empty value, which you use later to get the default value

Reply:
By the way, the description of the flag has a typo (`then`).
Being picky, I would rewrite it to something like `The minimum logging level; levels are, in ascending order: debug, info, warn, error"` based on the `sap_host_exporter` code